package.json

```diff
@@ -73,12 +74,13 @@
         "unlocalize": "rimraf src/nls && git checkout -- src/nls",
         "postinstall": "grunt install",
         "build": "grunt build-browser",
+        "watch:api": "grunt watch:bramble",
         "preproduction": "grunt build-browser-compressed && npm run unlocalize",
         "production": "npm start -- --gzip",
         "test": "grunt test && grunt build-browser-compressed",
         "server": "http-server -p 8000 --cors",
         "prestart": "npm run localize && grunt build-browser-dev",
-        "start": "npm run server || true",
+        "start": "run-p server watch:api || true",
```

Comment:
darn, I just noticed that the `|| true` doesn't really work well with `run-p` on mac. To fix that, let's break this into two different scripts:
```js
"start:server": "run-p server watch:api",
"start": "npm run start:server || true"
```